lightning/src/ln/channel.rs

```diff
@@ -557,6 +557,7 @@ pub const MIN_DUST_LIMIT_SATOSHIS: u64 = 330;
 /// channel_id in ChannelManager.
 pub(super) enum ChannelError {
 	Ignore(String),
+	Warn(String),
```

Comment:
Ah, glossed over https://github.com/rust-bitcoin/rust-lightning/pull/1011#issuecomment-885035137.

Reply:
The new closing fee range stuff requires sending `warning` messages, but I wanted to land this asap, vs warnings we're not in a rush. Thus, I slipped this in so that we can "send" warning messages before we land actual support for them.